k8s/demo/dbench/dbench-hostpath.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: dbench-hp
+spec:
+  template:
+    spec:
+      containers:
+      - name: dbench-hp
+        image: logdna/dbench:latest
+        imagePullPolicy: Always
+        env:
+          - name: DBENCH_MOUNTPOINT
```

Comment:
Is it possible to explain what each of these env variable is meant to do?
In other words yaml comments !!